index.js

```diff
@@ -27,6 +27,44 @@ exports.NativeScriptJsonpPlugin.prototype.apply = function (compiler) {
     });
 };
 
+exports.ExcludeUnusedElements = function () {
+    //
+}
+
+exports.ExcludeUnusedElements.prototype = {
+    apply: function (compiler) {
+        compiler.plugin("normal-module-factory", function (nmf) {
+            nmf.plugin("before-resolve", function (result, callback) {
+                if (!result) {
+                    return callback();
+                }
+
+                if (result.request === "globals" || result.request === "ui/core/view") {
+                    return callback(null, result);
+                }
+
+                if (result.context.indexOf("tns-core-modules") === -1) {
+                    if (result.contextInfo.issuer &&
+                        result.contextInfo.issuer.indexOf("element-registry") !== -1 && global["ELEMENT_REGISTRY"][result.request] &&
```

Comment:
`global["ELEMENT_REGISTRY"][result.request] && !global["ELEMENT_REGISTRY"][result.request]`

I think we need just `!global["ELEMENT_REGISTRY"][result.request]`